### PR TITLE
Ci update

### PR DESCRIPTION
**Pull request by luap-pik**
I am starting this PR to review our CI setup and look into the codecov problems.

### CODE REVIEW

**Comment by luap-pik** [comment]
Problem1: The old travisCI seems to be still running. Can we deactivate this?
Problem2: The doc build needs to be fixed, that should be straight-forward.

**Comment by SabineAuer** [comment]
@luap-pik: why do you want to get rid off travis completely and write the CI -yml yourself? I thought maybe moving to travis.org to travis.com could help?

**Comment by SabineAuer** [comment]
@luap-pik : I don't mind using the github internal CI. Only code coverage we should also include, right?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            ${{ runner.os }}-test-${{ env.cache-name }}-
+            ${{ runner.os }}-test-
+            ${{ runner.os }}-
+      - uses: julia-actions/julia-buildpkg@latest
+      - run: |
+          git config --global user.name Tester
+          git config --global user.email te@st.er
+      - uses: julia-actions/julia-runtest@latest
+        continue-on-error: ${{ matrix.version == '1.2' }}
+      - uses: julia-actions/julia-processcoverage@v1
```
Comment (luap-pik) [inline .github/workflows/ci.yml:43]: code coverage is actually included here

**Comment by luap-pik** [comment]
@SabineAuer I just wanted to try this out and see wether the tests and coverage results are more reliable. Also, the interface is nicely integrated in github when you click on checks.  I din't write the yml myself it is basically the default one and does the same as in the old travis.yml file.

**Comment by luap-pik** [comment]
an example: https://github.com/FHell/EmbeddedGraphs.jl

**Comment by SabineAuer** [comment]
@Paul: I see, looks like a good solution to me. Only I do not understand why the CI fails for Julia higher that 1.3. Do you?
Also the documentation CI is failing? What is it actually checking?

**Comment by luap-pik** [comment]
I think it is fine to merge

**Comment by SabineAuer** [comment]
Yes, I think so too. We can look at the code coverage limit later on.

**Pull request merged**